src/voloctree/voloctree_mapper.hpp

```diff
@@ -32,102 +32,41 @@
 #include <vector>
 #include <unordered_map>
 
+#include "volume_mapper.hpp"
 #include "bitpit_voloctree.hpp"
```

Comment:
@edoardolombardi Do not include "bitpit_voloctree.hpp" inside the voloctree module. Include the specific headers you need.